lua/neogit/popups/commit.lua

```diff
@@ -156,7 +156,19 @@ local function create()
         {
           key = "w",
           description = "Reword",
-          callback = function() end
+          callback = function()
+            a.dispatch(function ()
+              local msg = a.wait(cli.log.max_count(1).pretty('%B').call())
+              msg = vim.split(msg, '\n')
+
+              a.wait(prompt_commit_message(msg))
+              local _, code = a.wait(cli.commit.commit_message_file(COMMIT_FILE).amend.only.call())
```

Comment:
Is `only` required here?

Reply:
yes, otherwise the operation would fail without staged changes.
From the manpage:
```
If this option is specified together with --amend, then no paths need to be specified, which can be used to amend the last commit without committing changes that have already been staged. 
```